scripts/proxy.sh

```diff
@@ -0,0 +1,5 @@
+# edit the proxy details
+#
+# export http_proxy="http://10.0.2.5:3128"
+# export https_proxy="http://10.0.2.5:3128"
+# export no_proxy="localhost,cluster.local,127.0.0.1,::1,10.0.2.10,10.0.2.20,10.0.2.30"
```

Comment:
This is configuration rather than a script we want to execute to deploy some software. Let's move it into `config.example/` rather than `scripts/`.

Reply:
It can't be moved since setup.sh requires the use of proxyies as well to install software.